src/components/Navbar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Navbar = () => (
+  <nav className="navbar">
+    <h1 className="brand">
+      <a href="/">Math Magicians</a>
+    </h1>
+    <ul className="list">
+      <li className="list-items">
+        <a href="/">Home</a>
+      </li>
+      <li className="pipe">|</li>
+      <li className="list-items">
+        <a href="/calculator">Calculator</a>
+      </li>
+      <li className="pipe">|</li>
+      <li className="list-items">
+        <a href="/quote">Quote</a>
+      </li>
+    </ul>
+  </nav>
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 However, I suggest you to use the `<Link> and <NavLink>.` These are components provided by react-router-dom to navigate around the react application. Generally, we use anchor tags for this purpose while navigating. Then what’s the difference? Let’s find out.

- Anchor tags will reload the page and re-render all the components. In contrast, `<Link> and <NavLink>` will only re-render updated components matched with the URL path of the Route without reloading. It helps the Single-Page Applications to work faster while routing. If you need more information visit this link [What are the differences between NavLink, Link](https://www.educative.io/answers/what-are-the-differences-between-navlink-link-and-a-in-react)

Reply:
Thanks for the suggestion. I will do that right away